package.json

```diff
@@ -40,7 +40,7 @@
     "gh-pages": "3.2.3",
     "is-interactive": "2.0.0",
     "lerna-changelog": "1.0.1",
-    "netlify": "6.1.29",
+    "netlify": "7.0.1",
```

Comment:
This dependency can be removed entirely since deployments will now rely on [`netilify-cli`](https://www.npmjs.com/package/netlify-cli)'s deploy command instead of [`netlify`](https://www.npmjs.com/package/netlify)'s deprecated `.deploy()` method.

[Source: Netlify v7 breaking changes](https://github.com/netlify/js-client/releases/tag/v7.0.0)